src/app/modules/test-navigator/test-navigator.component.ts

```diff
@@ -156,14 +157,15 @@ export class TestNavigatorComponent implements OnInit, OnDestroy {
     }
   }
 
-  private async updateValidationMarkers(root: TestNavigatorTreeNode) {
+  public async updateValidationMarkers(root: TestNavigatorTreeNode) {
     await this.indexService.refresh();
     const markers = await this.validationMarkerService.getAllMarkerSummaries();
     this.log('received validation markers from server: ', markers);
     root.forEach((node) => {
         node.validation = markers.has(node.id) ? new ValidationMarkerSummary(markers.get(node.id)) : ValidationMarkerSummary.zero;
     });
     this.changeDetector.detectChanges();
+    this.messagingService.publish(WORKSPACE_MARKER_UPDATE, {});
```

Comment:
why not put the received markers on the bus (or something slightly more useful)?

Reply:
Thought about that, I guess it doesn't hurt. The filter bar has no use for the markers, though, as it has to traverse the tree and collect markers by file extension and filter state. I don't want to put the whole tree on the bus for this event.

Reply:
Thought about that, I guess it doesn't hurt. The filter bar has no use for the markers, though, as it has to traverse the tree and collect markers by file extension and filter state. I don't want to put the whole tree on the bus for this event.

Reply:
Thought about that, I guess it doesn't hurt. The filter bar has no use for the markers, though, as it has to traverse the tree and collect markers by file extension and filter state. I don't want to put the whole tree on the bus for this event.

Reply:
Thought about that, I guess it doesn't hurt. The filter bar has no use for the markers, though, as it has to traverse the tree and collect markers by file extension and filter state. I don't want to put the whole tree on the bus for this event.

Reply:
Thought about that, I guess it doesn't hurt. The filter bar has no use for the markers, though, as it has to traverse the tree and collect markers by file extension and filter state. I don't want to put the whole tree on the bus for this event.